_posts/The importance_of_being_earnest.md

```diff
@@ -0,0 +1,262 @@
+Library whereupon he was encouraged to open a
+Xerox Centre; it was sorely needed.
+
+The tidal waves of change he has seen “are not
+little, it has been change upon change”. He starts
+with how almost everything is now electronic
+and automated, rather than manual and walks us
+through the analog-to-digital revolution. “Every
+technology (from the glory days of the typewriter
+and liquid Xerox to those of floppy disks, CDs),
+every process has undergone a drastic shift.”
+
+Simply in terms of manpower, at the beginning,
+Ravichandran employed twelve to thirteen
+operators. This was required as the facilities were
+so primitive; something as simple as back to back
+printing had to be manually fed twice and
+inverted. Ten copies would have to be printed out
+manually as many times. “Technology has now
+advanced such that a single person can handle
+three to four machines- in fact I handle as many
+myself” he chortles.
+
+Then with a sombre expression, he says “As a
+result of all these advancements, a student’s
+thesis gets printed, bound and done with in a
+span of two hours. Back then, it would be a two
+month affair. A real relationship would develop
+with each student.”
+
+Even as he speaks, a student gives her thesis on a
+pen drive and asks him in broken Kannada if she
+can collect it the next day. He replies in the
+affirmative (in perfect English, might I add) and
+out she walks.
+
+He continues, the nostalgia heavy in his voice,
+“They (the students) would bring a couple of
+pages each day, handwritten. They would sit with
+me and dictate as I keyed it in, rectifying mistakes
+along the way. They would then go back and show the day’s draft to their professor, get it corrected.
+Then return the next day with edits and more
+pages to key in. Once the final copy was keyed in,
+they would get multiple Xeroxes of it.”
+
+He speaks of the trifling troubles they faced too:
+for the cover page of the thesis, they required
+“glossy” papers called dressing sheets that were
+only available with a vendor outside campus;
+getting the cover printed was a three day affair.
+Moreover, different fonts were not possible as all
+the typing was done with one typewriter; one
+need only look at old documents in the library
+archives for evidence.
+
+He then speaks of how mathematical symbols
+were printed, a slightly cumbersome matter to
+this day. It is in equal amounts both, appalling
+and amazing. In the typewriter, there would be
+a feature to insert different “wheels” containing
+various special symbols. A Dizzee wheel was one
+that contained mathematical symbols. So, for a
+simple ‘α’, one could either write it out manually everywhere it appeared (for brackets, this was the
+only option), or even more painstakingly: type it
+out multiple times with the wheel, cut each one of
+them out into tiny pieces and paste it everywhere
+it appeared.
+
+He then goes on to speak about typesetting,
+hailing LaTeX (“your generation is so privileged,
```

Comment:
btw you can use actual LaTeX in markdown (using `$` as usual)

Reply:
Yeh, thanks